### PR TITLE
Merge final code to the production branch

### PR DESCRIPTION
**Pull request by priscilamattos**
Merge final code to the production branch.

### CODE REVIEW

File: src/Components/SongCard.jsx [hunk 1]
```diff
@@ -1,115 +1,25 @@
...
-  }),
-}));
-
-const SongCard = () => {
-  const [expanded, setExpanded] = React.useState(false);
-
-  const handleExpandClick = () => {
-    setExpanded(!expanded);
+const SongCard = ({ song }) => {
+  console.log(song);
```
Comment (GigiPoon) [inline src/Components/SongCard.jsx:2]: remove console logs if not needed

File: src/Components/SearchBox.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <div className="searchBox wrapper">
+          <h2>Please select a song below:</h2>
+          <form className="searchForm" action="submit">
+            <select
+              value={selectedSongTitle}
+              onChange={(e) => {
+                setSelectedSongTitle(e.target.value);
+              }}
+            >
+              <option value="">Please select a song</option>
```
Comment (GigiPoon) [inline src/Components/SearchBox.jsx:14]: consider adding after your value a disable function 
<option value="
----> add that ---> " disabled="disabled"
> 
to disable user from selecting "Please select a song"

File: src/App.js [hunk 3]
```diff
@@ -23,19 +27,28 @@ function App() {
     });
   }, []);
 
+  useEffect(() => {
```
Comment (GigiPoon) [inline src/App.js:30]: can add comments here explaining what this useEffect does

**Review by GigiPoon — APPROVED** [review]

File: src/Components/SearchBox.jsx (hunk 2 above)
Comment (priscilamattos) [reply src/Components/SearchBox.jsx:14]: Done!

File: src/App.js (hunk 3 above)
Comment (priscilamattos) [reply src/App.js:30]: Comments added.

File: src/Components/SongCard.jsx (hunk 1 above)
Comment (priscilamattos) [reply src/Components/SongCard.jsx:2]: Removed.

**Comment by priscilamattos** [comment]
Final changes made as per request.

**Pull request closed without merging**